python/pyspark/sql/column.py

```diff
@@ -337,7 +337,8 @@ def cast(self, dataType):
             raise TypeError("unexpected type: %s" % type(dataType))
         return Column(jc)
 
-    astype = cast
+    astype = copy_func(cast)
+    astype.__doc__ = ":func:`astype` is an alias for :func:`cast`."
```

Comment:
Do we want to have versions for these API?

I think it's more clear to do this way:
```
@since(1.3)
def astype(self, dataType):
     """ Convert the column into type ``dataType``.
     
     :func:`astype` is an alias for :func:`cast`.
     """
     return self.cast(dataType)
```